File: src/main/java/ch/zxseitz/j3de/math/MathUtils.java
package ch.zxseitz.j3de.math;

public class MathUtils {
    public static final float TOLERANCE = 0.000001f;

    public static boolean isFloatEquals(float a, float b) {
        return Math.abs(a - b) < TOLERANCE;
    }

    private MathUtils() {}
}

File: src/main/java/ch/zxseitz/j3de/math/MathUtils.kt
package ch.zxseitz.j3de.math;

import kotlin.math.abs

class MathUtils private constructor() {
    companion object {
        const val TOLERANCE: Float = 0.000001f;
        fun isFloatEquals(a: Float, b: Float): Boolean {
            return abs(a - b) < TOLERANCE;
        }
    }
}
